src/test/java/uk/gov/justice/hmpps/casenotes/controllers/CreateCaseNoteIntTest.kt

```diff
@@ -136,6 +142,21 @@ class CreateCaseNoteIntTest : IntegrationTest() {
     hmppsEventsQueue.receivePersonCaseNoteEvent().verifyAgainst(PersonCaseNoteEvent.Type.CREATED, Source.DPS, saved)
   }
 
+  @Test
+  fun `can create a case note without caseload id when not sync to nomis`() {
+    val type = getAllTypes().first { !it.syncToNomis }
+    val request = createCaseNoteRequest(type = type.typeCode, subType = type.code)
+    val response = createCaseNote(personIdentifier(), request).success<CaseNote>(HttpStatus.CREATED)
+
+    val saved =
+      requireNotNull(noteRepository.findByIdAndPersonIdentifier(fromString(response.id), response.personIdentifier))
+    assertThat(saved.system).isEqualTo(System.DPS)
+
+    await withPollDelay ofSeconds(1) untilCallTo { hmppsEventsQueue.countAllMessagesOnQueue() } matches { it == 1 }
+    val event = hmppsEventsQueue.receivePersonCaseNoteEvent()
+    assertThat(event.eventType).isEqualTo("person.case-note.created")
```

Comment:
I'd check syncToNomis = false and source = DPS to confirm it won't be picked up by the DPS -> NOMIS sync